res/css/views/elements/_EventTilePreview.scss

```
@@ -0,0 +1,22 @@
+/*
+Copyright 2016 OpenMarket Ltd
```

```suggestion
Copyright 2021 Steffen Kolmer <steffen.kolmer@nordeck.net>
```

You can use your own licence :)

---

(or Matrix Foundation works fine too)